lib/pip.js

```diff
@@ -96,16 +96,23 @@ function generateRequirementsFile(
   }
 }
 
-function pipAcceptsSystem(pythonBin) {
+async function pipAcceptsSystem(pythonBin) {
   // Check if pip has Debian's --system option and set it if so
-  const pipTestRes = spawnSync(pythonBin, ['-m', 'pip', 'help', 'install']);
-  if (pipTestRes.error) {
-    if (pipTestRes.error.code === 'ENOENT') {
+  try {
+    const pipTestRes = await spawn(pythonBin, ['-m', 'pip', 'help', 'install']);
+    return (
+      pipTestRes.stdoutBuffer &&
```

Comment:
`stdoutBuffer` will be provided unconditionally, so no need to confirm on it

Reply:
Is that the same case with `stderrBuffer`?

Reply:
Yes, but in case of error handling, there could be an edge case, where the command fails because it cannot be invoked for some reason and then the error won't hold any of those properties